docs/hello.adoc

```diff
@@ -49,18 +51,18 @@ Below we backtrack what happened:
 
 The Hello component that rendered the actual page is very simple:
 
-.react4xp/components/Hello.tsx
+.react4xp/hello/components/Hello.tsx
 [source,TypeScript]
 ----
-include::../src/main/resources/react4xp/components/Hello.tsx[]
+include::{sourcedir}react4xp/components/hello/Hello.tsx[]
 ----
 
 It takes a single parameter, HelloProps. The type is declared like this:
 
 .types/HelloProps.ts
 [source,TypeScript]
 ----
-include::../src/main/resources/types/HelloProps.ts[]
+include::{sourcedir}types/HelloProps.ts[]
 ----
 
 The component is rendered because it has been added to the componentRegistry:
```

Comment:
Add a definition of Component registry:
`ComponentRegistry is a registry of react views used to render components by their type`